daemon/algod/api/algod.oas2.json

```diff
@@ -655,6 +655,12 @@
             "in": "path",
             "required": true
           },
+          {
+            "type": "boolean",
+            "description": "If true, only the block header (exclusive of payset or certificate) may be included in response.",
+            "name": "header-only",
```

Comment:
We use "exclude" in a couple other places. Maybe we should do the same here?  So either this could be `exclude-payset`, or we could add an "exclude" parameter that takes any of "certificate" "header" "payset"?

Reply:
Discussed: adding a comment in handler around ideal state in a V3.